### ISSUE TITLE
KCD, Bengaluru, India

### ISSUE THREAD

**Issue by nkhare** [cncf/kubernetes-community-days#169]
If you do not have your team assembled, please describe the location you're interested in organizing an event, proposed dates, and information about your availability and experience with hosting a conference. If you have additional organizers and volunteers, include their information here as well. Continue to update the issue when you add new organizers and sponsors until you are ready to commit to hosting, then complete the submission checklist below. 

### Organizer 1: First and Last Name, Company, Title, Email 
Neependra Khare, CloudYuga, Founder and Principal Consultant, neependra@cloudyuga.guru

Once you have identified your 3 organizers, 2 sponsors, please complete the information below and update your issue.

### Submission checklist:

*Please check each of these indicating that you have met these requirements:

* [x] All organizers signed off to the event guidelines and CNCF Code of Conduct.
* [x] Method to accept payments including sponsorships and payout expenses
* [x] One organizer is a CNCF Member employee, Ambassador, or CNCF Project Maintainer
* [x] Minimum of 3 organizers from 3 different organizations
* [x] No other Kubernetes Community Days event in the same city
* [x] Agree to use Eventbrite for registration
* [x] Agree to use Sched for organizing the program page

**Please include all the organizers following the format specified. At a minimum, you must have 3 organizers from 3 different organizations.**

### Organizer 1: First and Last Name, Company, Title, Email 
Neependra Khare, CloudYuga, Founder and Principal Consultant, neependra@cloudyuga.guru

### Organizer 2: First and Last Name, Company, Title, Email
Krishna Kumar, Huawei India, Chief Architect Cloud , krish00@gmail.com

### Organizer 3: First and Last Name, Company, Title, Email
Ajeet Singh Raina, Dell EMC, Principal Systems Development Engineer, ajeetraina@gmail.com

### Organizer 4:
Kiran Mova, Maya Data, Co-Founder, Chief Architect, kiran.mova@mayadata.io  

### Organizer 5:
Rajesh Challa, Samsung,  raj@samsung.com

### List of Sponsors:
- CloudYuga
- MayaData
- GrafanaLabs

**Comment by krishna-mk** [comment]
Superb! We are in for the KCD Bangalore 2020.

**Comment by christinevblum** [comment]
@krishna-mk Welcome! Be sure to check the first box once all your organizers have agreed event guidelines and CNCF Code of Conduct

**Comment by krishna-mk** [comment]
### Submission checklist:

*Please check each of these indicating that you have met these requirements:

* [x] All organizers signed off to the event guidelines and CNCF Code of Conduct.
* [x] Method to accept payments including sponsorships and payout expenses
* [x] One organizer is a CNCF Member employee, Ambassador, or CNCF Project Maintainer
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Each organizer needs to live in or just outside of the KCD location
* [x] No other Kubernetes Community Days event in the same city
* [x] Agree to use Eventbrite for registration
* [x] Agree to use Sched for organizing the program page

Yes agree to 
https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

**Comment by christinevblum** [comment]
The next step is to provide the names of your three sponsors. (Sorry, we left it off of the initial version of the issue template.) Once we get your 3 sponsor names your team will be signed off to start your event pages.

**Comment by aravindputrevu** [comment]
@christinevblum What are the requirements for being a sponsor? I work at Elastic and I could if there is a way to support the community & the event.

**Comment by KTan1226** [comment]
@aravindputrevu Please find the sponsorship guidelines here: https://kubernetescommunitydays.org/organizing-finding-sponsors/.

**Comment by kmova** [comment]
Awesome! MayaData is in!  Both as Organizing and being a sponsor.

**Comment by aravindputrevu** [comment]
Great! 

@nkhare & @krishna-mk! Let me know about this, I can talk internally to figure out someway.

**Comment by krishna-mk** [comment]
@aravindputrevu , let us meet quick next week (all Bangalore guys), after the Kubecon, and then have some sponsorship doc in place. May be one generic template will help :-)

**Comment by aravindputrevu** [comment]
@krishna-mk Sure. That'd be great. Let us catch up next week!

**Comment by rajibmitra** [comment]
+1 for this.

**Comment by in28minutes** [comment]
Hi. I'm Ranga. I would love to be part of this.

**Comment by krishna-mk** [comment]
Hi Ranga, Sorry for the late reply. We are making the plan and a list to get all volunteers in. Please hold on.

**Comment by in28minutes** [comment]
Awesome Krishna.

**Comment by nkhare** [comment]
@KTan1226 @iennae In one of the earlier thread Mayadata already confirmed as one of the sponsor. I, on behalf of CloudYuga, also confirms as being one of the sponsor. 

We have fixed the event date to be 27th June'2020 and finalised the venue as well. We need to pay some advance amount to hotel to block it by tomorrow. 

So we would like to request you confirm this event and close on dates.

**Comment by kmova** [comment]
Confirming MayaData as a sponsor.

**Comment by KTan1226** [comment]
Hi @nkhare- can you confirm the fiscal sponsor for the event (company that will accept payments)? Also, can you confirm that all organizers have read and signed off on the CNCF event guidelines and Code of Conduct? Once both items are confirmed, please share the CNCF Slack usernames for all organizers and I will add you to the  KCD organizers channel. Let me know if you have any questions. Thanks!

**Comment by cloudyuga** [comment]
> Hi @nkhare- can you confirm the fiscal sponsor for the event (company that will accept payments)? 
Cloudyuga Technologies Pvt. Ltd. 

> Also, can you confirm that all organizers have read and signed off on the CNCF event guidelines and Code of Conduct?
Yes. 

> Once both items are confirmed, please share the CNCF Slack usernames for all organizers and I will add you to the KCD organizers channel.
Will do

**Comment by nkhare** [comment]
I read and agree to 
https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

**Comment by ajeetraina** [comment]
I went through KCD code of conduct, read and agree to
https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

**Comment by kmova** [comment]
I agree and will comply with the CNCF event guidelines and code of conduct.  https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

kiranmova is my CNCF slack id.

**Comment by krishna-mk** [comment]
I read and agree to
https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

krishna-mk is my slack ID.

**Comment by aravindputrevu** [comment]
I read and agree to
https://kubernetescommunitydays.org/code-of-conduct/ and https://kubernetescommunitydays.org/organizing

aravind is my CNCF slack id.

**Comment by KTan1226** [comment]
Great, thanks! I've added all organizers to the KCD organizers channel. Please confirm your fiscal sponsor and then I can sign off on you creating your site.

**Comment by kmova** [comment]
CloudYuga,  @nkhare company - will be our fiscal sponsor.

**Comment by nkhare** [comment]
@KTan1226 We are all set. We have already signed the contract with the hotel and paid first installment. Please create the website asap, so we can get other things going.

**Comment by cloudyuga** [comment]
@KTan1226 Can you please add "neependra" in KCD organizers channel. Thanks

**Comment by KTan1226** [comment]
Great @nkhare! You are approved to build your site. You can find instructions on how to get started here: https://kubernetescommunitydays.org/organizing-creating-website/. Let me know if you have any questions.

**Comment by nkhare** [comment]
> Great @nkhare- one last thing: Can you please confirm your 3 sponsors for the event?
> 
> Once confirmed, I can sign off on you building your site.

@KTan1226  so far we have CloudYuga and Mayadata as two sponsors. We are discussing with few others and would should have 3rd in 7-10 days. We would start building the building the website in parallel from the template and once you sign off, we will go public quickly.

**Comment by KTan1226** [comment]
Hi, @nkhare- Thanks for the update. All sponsors need to be confirmed before you move forward with building your site.

**Comment by gouthamve** [comment]
@nkhare @kmova @KTan1226 I'd like to confirm Grafana Labs as a sponsor for KCD Bengaluru.

I've been in contact with one of the organisers (Kiran Mova @kmova) regarding the event and can confirm the silver level sponsorship for the event!

Thanks,
Goutham, Software Engineer @grafana and maintainer for @prometheus and @cortexproject

**Comment by nkhare** [comment]
thanks @gouthamve. @KTan1226  can we now take it forward and create the website. We are planning to make the site live before Kubernetes Forum, Bangalore; which is on 17th Feb'2020.

**Comment by KTan1226** [comment]
Great news @nkhare! You are approved to create your site and I will be in touch with your email alias. 

In the interim, can you share the CNCF Slack usernames for all organizers? I will add them to the KCD organizers channel. Also, do you have a  Twitter account for the event?  If so, please share the handle with me and we can help amplify.

**Comment by nkhare** [comment]
@KTan1226 We saw the test email from the email alias. Is it mandatory to use that for all the communication. We are already using other alias.  I think everyone got added in the KCD organizer channel. I'll double check. 

We have a twitter handle @KCDBengaluru .

**Comment by KTan1226** [comment]
Hi @nkhare- Yes, we are requiring organizers to use the created alias.

**Comment by nkhare** [comment]
Sure. @KTan1226 Can you please add @kmova  and Rajesh also in the mailing list. I have updated their details in the first comment.

**Comment by KTan1226** [comment]
Sure thing @nkhare- just updated! I will send a test email as well.

**Comment by nkhare** [comment]
@KTan1226 As discussed yesterday during the call that it would still take a few weeks for you to close the Eventbrite engagement, we can explore some other alternatives. I mentioned that we can use  [Instamojo](https://www.instamojo.com/) or [Razorpay](https://razorpay.com/), which are cheaper alternative to Eventbrite and take care of Indian Taxes as well. 

Given that we have the event coming up soon, we would like to use one of above and start the registration.  Let us know if you have any concerns on this?

**Comment by cloudyuga** [comment]
@KTan1226 Due to ongoing Covid-19 issue, we have decided to postpone the event. There is no future date as now. We should wait until things normalize and then take further decisions.

**Comment by nkhare** [comment]
@KTan1226 Due to ongoing Covid-19 issue, we have decided to postpone the event. There is no future date as now. We should wait until things normalize and then take further decisions.

**Comment by KTan1226** [comment]
Hi @nkhare. No problem- thanks for the update. Can we please update your website? I would recommend we change the date to Dec 1 temporarily and we can always change it later.

**Comment by nkhare** [comment]
Just for official log here. CloudYuga (My company) had to pay ~$2100 to postpone/cancel the event and new booking would a new process altogether.

**Comment by KTan1226** [comment]
Thanks for the update @nkhare. Can you update your KCD website? Do you have a new date in mind? We could always set it temporarily for March 2021 and update as you finalize.

**Comment by cloudyuga** [comment]
@KTan1226 All the organizers had recently recently visited this and we are planning to have the event on 5th and 6th February 2021.

**Comment by nkhare** [comment]
@KTan1226 All the organisers recently revisited this and we plan to have a virtual event on 5th and 6th February 2021.

**Comment by idvoretskyi** [comment]
@nkhare we currently don’t have a defined virtual KCD’s strategy for 2021. We’ll keep this thread updated.

**Comment by xmulligan** [comment]
@nkhare I'm currently working on the virtual strategy, do you have time for a chat sometime this week or next? https://calendly.com/billmulligan/1-1-with-bill-mulligan

**Comment by nkhare** [comment]
> @nkhare I'm currently working on the virtual strategy, do you have time for a chat sometime this week or next? I would love to help you with this! https://calendly.com/billmulligan/1-1-with-bill-mulligan

@xmulligan 
Sure. I have sent the invite for 14th Dec at 5 PM IST.  

@all
Let us know if anyone of you would like to join.

**Comment by xmulligan** [comment]
Thanks! Really looking forward to the conversation!

**Comment by nkhare** [comment]
> Thanks! Really looking forward to the conversation!

Google meet seems to down for me. Can you please join https://zoom.us/j/92981048770?pwd=aHlpcmtqZWF4U215NkhmSW1JdWl4QT09

**Comment by xmulligan** [comment]
Just saw this now. Tried messaging you on the CNCF slack first. Joining now

**Comment by xmulligan** [comment]
The event is now live https://community.cncf.io/events/details/cncf-kcd-bengaluru-presents-kubernetes-community-days-bengaluru/#/

**Comment by xmulligan** [comment]
@nkhare just wanted to check in with you. The event is about 8 weeks away. Is there anything you need from CNCF or do you have any questions about the platform?

**Comment by nkhare** [comment]
@xmulligan Things are shaping up and everyone in the team doing their best.  Would it be possible to do a blog about the keynote speakers, sessions etc on the CNCF blog? We'll schedule the demo with the Bevy team soon and discuss about out schedule to see if there would be any challenges.

**Comment by xmulligan** [comment]
@nishakm Great to hear! You can absolutely write a blog for the CNCF website. You can find all the guidelines here https://github.com/cncf/foundation/blob/master/blog-guidelines.md. I am also happy to review it for you if you are interested. Let me know if you need anything else 😃

**Comment by nishakm** [comment]
@xmulligan I think you tagged the wrong person :). I will mute this thread.

**Comment by xmulligan** [comment]
Sorry meant to tag @nkhare

**Comment by xmulligan** [comment]
This event was a great success with over 1671 attendees. Looking forward to the next edition!  💯
